various-receipes/db-optimization/docker-compose-mongo.yml

```diff
@@ -0,0 +1,13 @@
+version: '3.6'
+services:
+  database:
+    image: mongo:4
```

Comment:
No optimizations? I know it has a built-in in-mem engine, may be only commercial

Reply:
Couldn't find any CLI option for tuning, do you know some?
The in-mem engine is enterprise-only AFAIK

Reply:
@goldbergyoni I looked how [MongoDB Memory Server NPM package](https://www.npmjs.com/package/mongodb-memory-server) been implemented and I saw that he use the storage engine `ephemeralForTest` which been said in [this SO answer](https://stackoverflow.com/a/67110672/5923666) that:
> Note that `ephemeralForTest` storage engine doesn't support transactions and is therefore of limited use on MongoDB 4.0 and newer servers.

although there seems to be no support 

so after some searching I found that what you do is what has been suggested [here](https://jira.mongodb.org/browse/SERVER-38762?focusedCommentId=3204204&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-3204204)

Reply:
Thanks @rlucaton !